### ISSUE TITLE
Barrier Interferes with sleep on Windows 10

### ISSUE THREAD

**Issue by darkSKULL-labs** [debauchee/barrier#993]
**Describe the bug**
Barrier prevents computer from sleeping on windows 10.

**To Reproduce**
powercfg /requests

**Expected behavior**

Should allow sleep in configuration.

**Screenshots**

Please do not link to image hosting sites, as these can be ephemeral. Instead, attach them to the issue. 

**Desktop (please complete the following information):**

 - OS: [e.g. Windows]
- Barrier version [e.g 2.3.3]
Windows 10

**Additional context**

**Comment by dev-101** [comment]
I had the same problem, solved it with some setting tips and power management app [described in this article](https://tehnoblog.org/how-to-share-one-mouse-and-keyboard-between-multiple-computers/).

**Comment by yan12125** [comment]
Seems the same issue as #338. I can also confirm disabling the option "synchronize screen savers" allows the Windows client to go to sleep.

**Comment by hmoffatt** [comment]
I've disabled "synchronize screensavers", but the powercfg output suggests this will still stop the Windows 10 client from sleeping. My server is Linux.

```
>powercfg /requests
DISPLAY:
None.

SYSTEM:
[PROCESS] \Device\HarddiskVolume5\Program Files\Barrier\barrierc.exe
```

If I stop the server then this still doesn't go away until I stop the client. Is there a solution for this?

**Comment by mattbatt** [comment]
I had this problem, it turns out that I left "Wake on LAN" turned on on my host computer.  To replicate I stopped barrier on the clients, put the host to sleep, and made sure it stayed asleep, then I started the client.  My host started right up with "boot on LAN" turned on which makes sense really.

**Comment by josephtesfaye** [comment]
> Seems the same issue as #338. I can also confirm disabling the option "synchronize screen savers" allows the Windows client to go to sleep.

Is there a way to wake up a sleeping client with Barrier? I haven't found any. My client computer doesn't have a mouse or keyboard so I want to know if it's possible to wake it up using Barrier alone.

**Comment by farhansolodev** [comment]
@josephtesfaye I am not 100% certain but I've noticed that whenever I turn on the server on my laptop, my PC (client) eventually wakes up from sleep (it has WOL enabled). It could be a coincidence and just some other program doing it though.

**Comment by nbolton** [comment]
Please try Deskflow or Input Leap as Barrier is no longer in development.

https://github.com/deskflow/deskflow
https://github.com/input-leap/input-leap

If this is still an issue in those projects, we would appreciate a cross-post of this issue.

**Comment by hmoffatt** [comment]
Can these interwork with Barrier?
I use Barrier on Windows and Linux, and I await the next Debian release before I can use input-leap on Linux.

**Comment by nbolton** [comment]
Yeah, both are compatible with Barrier client/server so you can try on one of your computers.
